tutorials/roostats/Zbi_Zgamma.py

```diff
@@ -0,0 +1,52 @@
+## \file
+## \ingroup tutorial_roostats
+## \notebook -js
+## Demonstrate Z_Bi = Z_Gamma
+##
+## \macro_image
+## \macro_output
+## \macro_code
+##
+## \date July 2022
+## \authors Artem Busorgin, Kyle Cranmer and Wouter Verkerke (C++ version)
+
+import ROOT
+
+# Make model for prototype on/off problem
+# Pois(x | s+b) * Pois(y | tau b )
+# for Z_Gamma, use uniform prior on b.
+w1 = ROOT.RooWorkspace("w", True)
```

Comment:
```suggestion
w1 = ROOT.RooWorkspace("w")
```
This second argument to enable the CINT export should not be used anymore in ROOT 6. can you remove it please, and also do the same in the C++ tutorial while at it? Thank you!